backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -215,8 +216,11 @@ trait StandardAsyncExecutionActor extends AsyncBackendJobExecutionActor with Sta
         |(
         |cd $cwd
         |INSTANTIATED_COMMAND
-        |)
+        |) > $stdoutPath 2> $stderrPath
         |echo $$? > $rcTmpPath
+        |# Still need to dump to stdout and stderr for PAPI
+        |cat $stdoutPath
+        |>&2 cat $stderrPath
```

Comment:
yeah ugh

Reply:
Hmm, at the back of my memory is that PAPI might have a special field for specifying `stdout` and `stderr`, and we use that to allow the auto-magical "real-time-stdout"ing

Reply:
Why do I have this sinking feeling that you're right. 😦 I may be able to get around that with `tee` but I was hoping not to have to introduce that dependency. I'll look into this more.